### PR TITLE
Feature/upgrade ci runners

### PR DESCRIPTION
**Pull request by JaviIG**
This PR updates the CI runners to node 16 and fixes a failing test in node16

EX-6907

### CODE REVIEW

**Review by tajespasarela — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: cypress/browsers:node14.17.0-chrome91-ff89
+    container:
+      image: cypress/browsers:node16.16.0-chrome105-ff104-edge
+      options: --user 1001
```
Comment (LuisMartinez15) [inline .github/workflows/build.yml:11]: Why do we need to add this `options: --user 1001` parameter? I didn't find anything related to this in the cypress documentation 😅

**Review by LuisMartinez15 — APPROVED** [review]
good job! thanks!

File: .github/workflows/build.yml (hunk 1 above)
Comment (LuisMartinez15) [reply .github/workflows/build.yml:11]: Nvm, I've found it! In case anyone also wonders why, [read](https://github.com/cypress-io/github-action#firefox).

Comment (JaviIG) [reply .github/workflows/build.yml:11]: I still don't understand why it is necessary. That's the only reference I found in their docs, asides from some issues[ like this one](https://github.com/cypress-io/cypress-docker-images/issues/560). We are not using Firefox yet, but seems that the image lacks certain permissions when installing?

[This is the error](https://github.com/empathyco/x/actions/runs/3131223359/jobs/5082443562) that gave without that user.
image

**Review by LuisMartinez15 — APPROVED** [review]

**Pull request merged**